homeassistant/components/lgthinq/entity_helpers.py

```diff
@@ -0,0 +1,3147 @@
+"""LG ThinQ entity descriptions and mapping table."""
+
+from __future__ import annotations
+
+from collections.abc import Collection
+from dataclasses import dataclass
+from datetime import time
+from enum import StrEnum, unique
+import logging
+from typing import Any, Generic, TypeVar
+
+from thinqconnect.const import DeviceType
+from thinqconnect.thinq_api import ThinQApiResponse
+
+from homeassistant.components.binary_sensor import BinarySensorEntityDescription
+from homeassistant.components.button import ButtonEntityDescription
+from homeassistant.components.climate import ClimateEntityDescription
+from homeassistant.components.event import EventEntityDescription
+from homeassistant.components.fan import FanEntityDescription
+from homeassistant.components.humidifier import (
+    HumidifierDeviceClass,
+    HumidifierEntityDescription,
+)
+from homeassistant.components.number import (
+    NumberDeviceClass,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.components.select import SelectEntityDescription
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.components.switch import SwitchEntityDescription
+from homeassistant.components.text import TextEntityDescription
+from homeassistant.components.time import TimeEntityDescription
+from homeassistant.components.vacuum import StateVacuumEntityDescription
+from homeassistant.components.water_heater import WaterHeaterEntityEntityDescription
```

Comment:
As you mentioned, we already limited a real functionality to a single platform.
But because we support almost 30 types of devices and 10 entities. (Home Appliances).
Limiting every code into a single platform per PR is very complicated based on our code.
And it will slow us down so much.
This file is mapping entities to devices.
Could you review this file in the last PR?